sklearn/decomposition/tests/test_pca.py

```diff
@@ -529,6 +529,50 @@ def test_pca_score3():
     assert_true(ll.argmax() == 1)
 
 
+def test_pca_score_with_different_solvers():
+    digits = datasets.load_digits()
+    X_digits = digits.data
+
+    svd_solvers = ['full', 'arpack', 'randomized']
```

Comment:
@lesteve How about using the global variable solver_list here to make the test consistent with other tests?